relay-event-schema/src/protocol/contexts/device.rs

```diff
@@ -161,6 +162,22 @@ pub struct DeviceContext {
     /// Whether location support is available on the device.
     pub supports_location_service: Annotated<bool>,
 
+    /// Width of the screen in pixels.
+    #[metastructure(pii = "maybe")]
+    pub screen_width_pixels: Annotated<u64>,
+
+    /// Height of the screen in pixels.
+    #[metastructure(pii = "maybe")]
+    pub screen_height_pixels: Annotated<u64>,
+
+    /// Locale of the device.
```

Comment:
Let's try to add more information to the doc comments, such as what is the format. I'll pick this up in my follow-up.